clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -714,7 +765,8 @@ static void VisitAccessorWrapper(CXXRecordDecl *Owner, ParentTy &Parent,
                                  CXXRecordDecl *Wrapper,
                                  Handlers &... handlers) {
   (void)std::initializer_list<int>{(handlers.enterStruct(Owner, Parent), 0)...};
-  VisitAccessorWrapperHelper(Wrapper, Wrapper->bases(), handlers...);
+  // FIXME base classes are not supported yet
+  // VisitAccessorWrapperHelper(Wrapper, Wrapper->bases(), handlers...);
```

Comment:
We still have to do this for offsets.

Reply:
Without base class support for the rest, dealing with base classes for just integration header offsets is pointless. Base class support can be added together, later.

Reply:
I disagree.  Do not disable functioning code.

Reply:
Are you saying that integration headers generated for inputs containing base classes are correct? I got the impression that the current code does not to cause any tests to fail only because the tests have been clipped back to avoid checking examples with base classes.

Reply:
The base class offsets should be right.  Someone else has an AR to create a test to validate them, but I don't see the bug report off hand.